patches/tModLoader/Terraria/ModLoader/ModLoader.cs

```diff
@@ -252,7 +265,55 @@ private static bool Unload()
 			}
 		}
 
-		private static void do_Unload()
+		[MethodImpl(MethodImplOptions.NoInlining)]
+		private static void Do_Unload() {
+			// Initialize a list of errors
+			var excs = new List<Exception>();
+
+			// Unload content last
+			PreUnload += Mods_Unload;
```

Comment:
this looks leftover, there's only ever one item in the invocation list.

Reply:
It was.